draft-ietf-tls-exported-authenticator.md

```diff
@@ -467,22 +472,28 @@ When validating an authenticator, a constant-time comparison should be used.
 ## Update of the TLS ExtensionType Registry
 
 IANA is requested to update the entry for server_name(0) in the registry for
-ExtensionType (defined in {{!TLS13}}) by replacing the value in the "TLS 1.3"
+ExtensionType (defined in {{!RFC8446}}) by replacing the value in the "TLS 1.3"
 column with the value "CH, EE, CR" and this document in the "Reference" column.
 
+The addition of "CR" to this column does not authorize the use of this extension
+in authenticator CertificateRequests or handshake CertificateRequests, only
+ClientCertificateRequest created as part of client-generated certificate requests.
```

Comment:
```suggestion
ClientCertificateRequest created as part of client-generated authenticator requests.
```

Reply:
(To try and make it more clear that this only pertains to this draft, and not the main TLS handshake.)